pkg/cloud/services/networking/network.go

```diff
@@ -102,23 +102,23 @@ func (s *Service) ReconcileExternalNetwork(openStackCluster *infrav1.OpenStackCl
 	case 0:
 		// Not finding an external network is fine
 		openStackCluster.Status.ExternalNetwork = &infrav1.NetworkStatus{}
-		s.scope.Logger().Info("No external network found - proceeding with internal network only")
+		s.scope.Logger().V(2).Info("No external network found - proceeding with internal network only")
 		return nil
 	case 1:
 		openStackCluster.Status.ExternalNetwork = &infrav1.NetworkStatus{
 			ID:   networkList[0].ID,
 			Name: networkList[0].Name,
 			Tags: networkList[0].Tags,
 		}
-		s.scope.Logger().Info("External network found", "network id", networkList[0].ID)
+		s.scope.Logger().V(2).Info("External network found", "id", networkList[0].ID)
 		return nil
 	}
 	return fmt.Errorf("found %d external networks, which should not happen", len(networkList))
 }
 
 func (s *Service) ReconcileNetwork(openStackCluster *infrav1.OpenStackCluster, clusterName string) error {
 	networkName := getNetworkName(clusterName)
-	s.scope.Logger().Info("Reconciling network", "name", networkName)
+	s.scope.Logger().V(2).Info("Reconciling network", "name", networkName)
```

Comment:
@anastaruno put these at log level 3 in e.g. https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/d84775f0cc9edc28e67cec40ca2cd194b294698a/pkg/cloud/services/loadbalancer/loadbalancer.go#L51

I think I like that because this feels like context information rather than essential information.